source/jormungandr/jormungandr/tests/utils_test.py

```diff
@@ -314,9 +315,34 @@ def test_read_best_boarding_positions():
         writer.writerow(["b", "a", "front"])
 
     d = read_best_boarding_positions(file_name)
-    assert d[make_best_boarding_position_key("a", "b")] == {
+    assert d[make_origin_destination_key("a", "b")] == {
         response_pb2.FRONT,
         response_pb2.BACK,
         response_pb2.MIDDLE,
     }
-    assert d[make_best_boarding_position_key("b", "a")] == {response_pb2.FRONT, response_pb2.BACK}
+    assert d[make_origin_destination_key("b", "a")] == {response_pb2.FRONT, response_pb2.BACK}
+
+
+def test_read_od_allowed_ids():
+    import shortuuid
+
+    file_name = '{}.csv'.format(shortuuid.uuid())
+
+    with open(file_name, 'w+') as file:
```

Comment:
Minor: You can write to a temporary file, no ?

Ex:
```
import tempfile
with tempfile.TemporaryDirectory() 
```

Reply:
I tried but didn't succeed ..